### DISCUSSION TITLE
Updates to spacy-loggers package not published to PyPI

### DISCUSSION THREAD

**Discussion by tpanza** [explosion/spaCy#12960] [Help: Other Questions] [answered]
Hello, the last version of `spacy-loggers` was published to the PyPI was [1.0.4 on December 8, 2022](https://pypi.org/project/spacy-loggers/#history). However, since then, there apparently have been changes to the spacy-loggers `main` branch but no subsequent releases published to PyPI.

What would it take to resolve this?

Right now, when you land on the main spacy-loggers GitHub page and start following the README, it defaults to showing docs and examples from the `main` branch, which has the unpublished changes. This suggests to readers to use `spacy.MLflowLogger.v2` in an [example config](https://github.com/explosion/spacy-loggers#example-config-1). This will fail because `spacy.MLflowLogger.v2` was implemented _after_ the 1.0.4 release.

Can we please get a publication of the latest `spacy-loggers` package to the PyPI?

**Comment by svlandeg** [comment, marked as answer]
Thanks for the note, this has been an oversight on our end. Apologies! We'll get that release out, this week or the next.

**Reply by adrianeboyd** [reply]
As an update, `spacy-loggers` v1.0.5 has now been published on pypi.
